### PR TITLE
Update auto_tagging.yml

### PR DESCRIPTION
**Pull request by samarthdd**
adding label as tag name on merging to branch

### CODE REVIEW

File: .github/workflows/auto_tagging.yml [hunk 1]
```diff
@@ -93,10 +95,35 @@ jobs:
...
         uses: actions/create-release@latest
         env:
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          GITHUB_TOKEN: ${{ secrets.GITHUBPAT }}
         with:
           tag_name: ${{ steps.tagging.outputs.tag_id }}
           release_name: ${{ steps.tagging.outputs.tag_id }}
           draft: false
           prerelease: ${{ steps.tagging.outputs.prerelease }}
+      - name : tag PR on merge
```
Comment (DinisCruz) [inline .github/workflows/auto_tagging.yml:104]: @samarthdd can describe in detail what is the sequence of events that are supposed to happen?

File: .github/workflows/auto_tagging.yml [hunk 2]
```diff
@@ -93,10 +95,35 @@ jobs:
...
+        run: |
+          if [[ "${{ github.event.pull_request.merged }}" == "true" ]] ; then
+          
+                  tag_name=${{ steps.tagging.outputs.tag_id }}
```
Comment (DinisCruz) [inline .github/workflows/auto_tagging.yml:108]: what is value?

is it the tag name of the PR comment?

If so when does this kick in, after the tag is added?

File: .github/workflows/auto_tagging.yml [hunk 3]
```diff
@@ -93,10 +95,35 @@ jobs:
...
+                  echo $tag_name
+                  
+                  color="$(openssl rand -hex 3)"
```
Comment (DinisCruz) [inline .github/workflows/auto_tagging.yml:111]: is this going to assign a random color?

if so, we need to change this to make sure the color is always the same (note how at the moment we only use one color)

File: .github/workflows/auto_tagging.yml [hunk 4]
```diff
@@ -93,10 +95,35 @@ jobs:
...
+                  echo $color
+
+                  pull_number=$(jq --raw-output .pull_request.number "$GITHUB_EVENT_PATH")
+                  echo $pull_number
+
+                  curl -X POST  -H "Accept: application/vnd.github.v3+json" -H"authorization: Bearer ${{ secrets.GITHUBPAT }}" https://api.github.com/repos/samarthdd/git-wiki-action/labels -d '{"name": "'"$tag_name"'" ,"color": "'"$color"'"}'
```
Comment (DinisCruz) [inline .github/workflows/auto_tagging.yml:117]: is `https://api.github.com/repos/samarthdd/git-wiki-action/labels` the right repo

**Review by DinisCruz — CHANGES_REQUESTED** [review]

File: .github/workflows/auto_tagging.yml (hunk 3 above)
Comment (samarthdd) [reply .github/workflows/auto_tagging.yml:111]: YES THIS is random color now. ok i will change to one color then

File: .github/workflows/auto_tagging.yml (hunk 4 above)
Comment (samarthdd) [reply .github/workflows/auto_tagging.yml]: fixed

File: .github/workflows/auto_tagging.yml (hunk 3 above)
Comment (samarthdd) [reply .github/workflows/auto_tagging.yml]: changed to color="D2C3F9" which is used in other PR

File: .github/workflows/auto_tagging.yml [hunk 5]
```diff
@@ -6,9 +6,11 @@ name: Auto tagging and release
 on:
   # Triggers the workflow on push or pull request events but only for the main branch
   push:
-    branches:
-      - main
-      - develop
+#     branches:
+#       - main
+#       - develop
```
Comment (DinisCruz) [inline .github/workflows/auto_tagging.yml:11]: Hi, shouldn’t we still limit this to these two branches, what happens if somebody does a PR into another branch?

**Review by DinisCruz — APPROVED** [review]

File: .github/workflows/auto_tagging.yml (hunk 5 above)
Comment (samarthdd) [reply .github/workflows/auto_tagging.yml:11]: Yes currently even on PR, i am checking only for develop and main branches. if PR is merged to some branch, it wont execute anything.

The run might fail while labelling PR. But I will check and verify once

Comment (samarthdd) [reply .github/workflows/auto_tagging.yml:11]: ok there is some issue. I willl check this

**Pull request merged**